indexer-app/src/cmr/indexer/data/concepts/service.clj

```diff
@@ -46,6 +48,7 @@
        :deleted deleted
        :service-name service-name
        :service-name-lowercase (string/lower-case service-name)
+       :service-type-lowercase (string/lower-case service-type)
```

Comment:
Why is it that you have only lowercase for the service-type, while service-name have both the lower-case and the normal service-name?

Reply:
service-type is controlled vocabulary and we will never have an instance where Harmony and harmony are distinct different values. I think that it takes up more resources to have both and it is not necessary.